.circleci/config.yml

```diff
@@ -1,72 +1,39 @@
 version: 2.1
-
+setup: true
```

Comment:
This is making the CI fail on `master`. Could you look into it @fotos ?

Reply:
Will do tomorrow (CET).

Reply:
Indeed having `setup: true` allows [only one workflow to defined](https://support.circleci.com/hc/en-us/articles/360060934851-Troubleshooting-Error-Message-Max-number-of-workflows-exceeded).

Funnily enough I did test the `configuration` before I said it's ready to merge:

```sh
$ circleci config validate
Config file at .circleci/config.yml is valid.
```

OK, I'll refactor this and open a new PR.

Reply:
Follow up PR is #182.